pkg/goDB/DBWorkManager.go

```diff
@@ -357,9 +391,9 @@ func (w *DBWorkManager) readBlocksAndEvaluate(workload DBWorkload, resultMap has
 				// Populate comparison value for current entry
 				if query.hasCondSip {
 					if isIPv4 {
-						comparisonValue.PutSipV4(sipBlocks[i*4 : i*4+4])
+						comparisonValue.PutSip(sipBlocks[i*4 : i*4+4])
```

Comment:
Why does `PutSip` only have one function signature, but `PutDip` has `V4`/`V6`?

Reply:
Because the source IP is the first thing that is stored in the key slice (so it's always at position `0`, no matter if it's IPv4 / IPv6)...